sirepo/package_data/static/js/cloudmc.js

```diff
@@ -71,15 +71,20 @@ SIREPO.app.config(() => {
         <div data-ng-switch-when="MaterialValue" data-ng-class="fieldClass">
           <div data-material-list="" data-model="model" data-field="field"></div>
         </div>
-        <div data-ng-switch-when="TallyList" data-ng-class="fieldClass">
+        <div data-ng-switch-when="PlotTallyList" data-ng-class="fieldClass">
           <div class="input-group">
             <select class="form-control" data-ng-model="model[field]" data-ng-options="t.name as t.name for t in model.tallies"></select>
           </div>
         </div>
+        <div data-ng-switch-when="SettingsTallyList" data-ng-class="fieldClass">
+          <div class="input-group">
+            <select class="form-control" data-ng-model="model[field]" data-ng-options="t as t.name for t in appState.models.settings.tallies track by t.name"></select>
```

Comment:
Either replace a null value with the first tally or add an "empty" option e.g.
```html
<select class="form-control" data-ng-model="model[field]" data-ng-options="t as t.name for t in appState.models.settings.tallies track by t.name">
    <option value="" disabled selected>Select a tally</option>
</select>
```
If the latter, all such dropdowns should have a similar option.

Reply:
I've added a proper tallyList directive now which will set the value if no item is selected. c866f2a